src/applications/check-in/day-of/tests/e2e/pages/Confirmation.js

```diff
@@ -97,12 +97,6 @@ class Confirmation {
       );
   };
 
-  validateExtraFooterMessage = () => {
```

Comment:
Why was this test removed?

Reply:
"day-of-travel-extra-message" no longer exists since the help block refactored some of this code into a single help block component, and it was asked that some of the extra content be removed in the process as well in the ticket